org.chessmd.chessmd.yml

```diff
@@ -0,0 +1,72 @@
+id: org.chessmd.chessmd
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: start.sh
+
+finish-args: 
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --filesystem=host
+  - --device=dri
+  
+modules: 
+  - name: chessmd
+    buildsystem: simple
+    
+    sources:
+      - type: file
+        path: app/ChessMD
+      - type: file
+        path: app/start.sh
+      - type: file
+        path: app/org.chessmd.chessmd.png
+      - type: file
+        path: app/org.chessmd.chessmd.desktop
+      - type: file
+        path: app/engine/stockfish17.1-popcnt
+      - type: file
+        path: app/opening/openings.bin
+      - type: file
+        path: app/opening/openings.headers
+      - type: file
+        path: app/lib/libjpeg.so.8
+      - type: file
+        path: app/lib/libtiff.so.5
+      - type: file
+        path: app/lib/libicui18n.so.70
+      - type: file
+        path: app/lib/libicuuc.so.70
+      - type: file
+        path: app/lib/libjbig.so.0
+      - type: file
+        path: app/lib/libdeflate.so.0
+      - type: file
+        path: app/lib/libicudata.so.70
+ 
+    build-commands:
+      - mkdir -p /app/bin
+      - install -Dm755 ChessMD /app/bin/ChessMD
```

Comment:
This should be built from source.

https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source

Reply:
this wasn't the question asked.

See https://github.com/flathub/flathub/pull/6960#issuecomment-3321749471